index.html

```diff
@@ -2,8 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>JS Documentation</title>
-    <link rel="stylesheet" href="style.css">
+    

```

Comment:
add link tag to link your css file with HTML as you have CSS file but it is not linked with HTML

Reply:
-  [ ] You did great giving a review and also using GitHub's inline comments feature. :+1: :100: According to the article on [How to communicate](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) when giving a review, Your review comment should be a suggestion instead of making it sound like an order. I'll suggest you edit these comments and rephrase your review. :smile: :wave: 

> **Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?**